test/snippet/io/sequence_file/sequence_file_output_fields_trait_1.cpp

```diff
@@ -15,18 +16,25 @@ int main()
 
     seqan3::sequence_file_output fout{std::ostringstream{},
                                       seqan3::format_fasta{},
-                                      seqan3::fields<seqan3::field::id, seqan3::field::seq_qual>{}};
+                                      seqan3::fields<seqan3::field::id, seqan3::field::seq, seqan3::field::qual>{}};
 
     for (int i = 0; i < 5; i++)
     {
         std::string id{"test_id"};
         // vector of combined data structure:
-        std::vector<seqan3::qualified<seqan3::dna5, seqan3::phred42>> seq_qual{{'N'_dna5, '7'_phred42}};
+        std::vector<seqan3::qualified<seqan3::dna5, seqan3::phred42>> seq_qual{{'N'_dna5, '7'_phred42},
+                                                                               {'A'_dna5, '1'_phred42},
+                                                                               {'C'_dna5, '3'_phred42}};
+
+        auto view_on_seq  = seqan3::views::get<0>(seq_qual);
+        auto view_on_qual = seqan3::views::get<1>(seq_qual);
 
         // ...
 
-        fout.emplace_back(id, seq_qual);       // note also that the order the argumets is now different, because
-        // or:                                    you specified that ID should be first in the fields template argument
-        fout.push_back(std::tie(id, seq_qual));
+        fout.emplace_back(id, view_on_seq, view_on_qual); // Also note that the order of the arguments is now
+                                                          // different, because you specified that ID should be
+                                                          // first in the fields template argument
+        // or:
+        fout.push_back(std::tie(id, view_on_seq, view_on_qual));
     }
```

Comment:
I think this snippet really confusing. So we declare an `seqan3::sequence_file_output` of type `seqan3::format_fasta` which only includes `id` and `sequence`. I explicit declare that I want to have `field::qual` which doesn't really mean anything, since `format_fasta` will drop this information?

The snippet itself talks about how the `order of the arguments is now different` but I actually don't know what its talking about. It is the same order as everywhere else ?!

I don't think I understand what this snippet is supposed to demonstrate. Any thoughts on this
@marehr @eseiler ?

Reply:
This should probably be fastq output and not fasta.

The ordering refers to the default `seq, id, qual` which should be more clear. (2nd snippet in https://docs.seqan.de/seqan/3-master-user/tutorial_sequence_file.html#autotoc_md213 shows the default order).

Reply:
Thx, I didn't get that the default order is different. Also fastq makes a lot more sense. I will change it.